data-raw/process_data_sets_starry.R

```diff
@@ -9,11 +10,160 @@ colnames(cabinet_turnover) <- colnames(cabinet_turnover) %>%
   str_replace_all(" ", "_")
 
 cabinet_turnover <- cabinet_turnover %>% 
-  mutate(start = as.Date(start), 
-         end = as.Date(end), 
+  mutate(start = as.Date(start, "%m/%d/%y"), # change format
+         end = as.Date(end, "%m/%d/%y"), # change format 
          president = as.factor(president), 
          position = as.factor(position),
          length = as.numeric(length), 
          days = as.numeric(days))
 
-devtools::use_data(cabinet_turnover, overwrite = TRUE)
\ No newline at end of file
+devtools::use_data(cabinet_turnover, overwrite = TRUE)
+
+# august-senate-polls ---------------------------------------------------------------
+august_senate_polls <- read_csv("data-raw/august-senate-polls/august_senate_polls.csv")
+colnames(august_senate_polls) <- colnames(august_senate_polls) %>% 
+  tolower() %>% 
+  str_replace_all(" ", "_")
+august_senate_polls <- august_senate_polls %>% 
+  mutate(
+    cycle = as.numeric(cycle), 
+    state = as.factor(state), 
+    senate_class = as.factor(senate_class),
+    start_date = as.Date(start_date), 
+    end_date = as.Date(end_date))
+
+devtools::use_data(august_senate_polls, overwrite = TRUE)
+
+# endorsements ---------------------------------------------------------------
+endorsements_2020 <- read_csv("https://projects.fivethirtyeight.com/endorsements-2020-data/endorsements-2020.csv")
+colnames(endorsements_2020) <- colnames(endorsements_2020) %>% 
+  tolower() %>% 
+  str_replace_all(" ", "_")
+endorsements_2020 <- endorsements_2020 %>% 
+  mutate(
+    position = as.factor(position), 
+    city = as.factor(city),
+    state = as.factor(state), 
+    endorser_party = as.factor(endorser_party),
+    category = as.factor(category)
+  )
+
+devtools::use_data(endorsements_2020, overwrite = TRUE)
+
+# forecast-review ---------------------------------------------------------------
+forecast_results_2018 <- read_csv("data-raw/forecast-review/forecast_results_2018.csv")
+colnames(forecast_results_2018) <- colnames(forecast_results_2018) %>% 
+  tolower() %>% 
+  str_replace_all(" ", "_")
+forecast_results_2018 <- forecast_results_2018 %>% 
```

Comment:
Can you shorten the variable names `democrat_winprobability` to `dem_win_prob`. Same for republicans?